docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,255 @@
+    │ cloud-node# kubeedge cloud init <arguments>              │
+    └──────────────────────────────────────────────────────────┘
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ On the second machine:                                   │
+    ├──────────────────────────────────────────────────────────┤
+    │ edge-node# kubeedge node join <arguments>                │
+    └──────────────────────────────────────────────────────────┘
+
+    You can then repeat the second step on as many other machines as you like.
+
+Usage:
+  kubeedge [command]
+
+Available Commands:
+  cloud       Cloud component command option for KubeEdge
+  node        Edge component command option for KubeEdge
+  version     Displays KubeEdge release version and code commit id.
+
+Flags:
+  -h, --help   help for kubeedge
+
+Use "kubeedge [command] --help" for more information about a command.
+ubuntu#
+```
+
+### kubeedge cloud --help
+
+```
+ubuntu# kubeedge cloud --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge cloud [command]
+
+Example usage:
+<Apt information to be added while implementation>
+
+Available Commands:
+  init        Bootstraps cloud component. Checks and install (if required) the pre-requisites.
+  reset       Teardowns cloud component.
+
+Flags:
+  -h, --help   help for cloud
+
+Use "kubeedge cloud [command] --help" for more information about a command.
+```
+
+### kubeedge cloud init --help
+
+```
+kubeedge cloud init --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge cloud init [flags]
+
+Example usage:
+<Apt information to be added while implementation>
+
+Flags:
+  -h, --help                        help for init
+      --kubeedge-version   string   use this key to download and use the required KubeEdge version (Optional, default will be Latest)
+      --kubernetes-version string   use this key to download and use the required Kubernetes version (Optional, default will be Latest)
+      --docker-version     string   use this key to download and use the required Docker version (Optional, default will be Latest)
+```
+
+### kubeedge cloud reset --help
+
+```
+kubeedge cloud reset --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge cloud reset
+
+Example usage:
+<Apt information to be added while implementation>
+
+Flags:
+  -h, --help   help for reset
+  
+```
+
+### kubeedge node join --help
+
+```
+kubeedge node join --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge node join [flags]
+
+Example usage:
+<Apt information to be added while implementation>
+
+Flags:
+  -h, --help                        help for join
+      --certPath           string   downloaded path of the certifcates generated by cloud component in this host (Mandatory)
+      --docker-version     string   use this key to download and use the required Docker version (Optional, default will be Latest)
```

Comment:
see similar comment for deciding on default versions, maybe some k8s version needs docker version x.x and above ?

Reply:
Yes, We will take of the supported and stable version of docker and K8S